### PR TITLE
Phoenix

### PR DESCRIPTION
**Pull request by pcasal**
If your PR is related to a contribution to the taxonomy, please, fill
out the following questionnaire. If not, replace this whole text and the
following questionnaire with whatever information is applicable to your PR.

**Describe the contribution to the taxonomy**

- ...
- ...
- ...

**Input given at the prompt**

```
   ...
```

**Response from the original model**

```
  ...
```

**Response from the fine-tuned model**

```
  ...
```

**Contribution checklist**

- [ ] The contribution was tested with `ilab generate`
- [ ] No errors or warnings were produced by `ilab generate`
- [ ] All [commits are signed off](https://github.com/instructlab/taxonomy/blob/main/CONTRIBUTING.md#legal) (DCO)
- [ ] The `qna.yaml` file contains at least 5 `seed_examples`
- [ ] The `qna.yaml` file was [linted](https://yamllint.com) and [prettified](https://onlineyamltools.com/prettify-yaml) ([yaml-validator](https://jsonformatter.org/yaml-validator) can do both)
- [ ] An `attribution.txt` file in the same folder as the `qna.yaml` file
- [ ] Content does not include PII or otherwise sensitive or confidential information
- [ ] Content does not include anything documented in the project's [Avoid these Topics](https://github.com/instructlab/taxonomy/blob/main/docs/SKILLS_GUIDE.md#avoid-these-topics) guidelines

### CODE REVIEW

**Comment by pcasal** [comment]
This is a contribution to the Phoenix Constellation knowledge - https://github.com/instructlab/taxonomy/issues/830

File: knowledge/astronomy/constellations/Phoenix/qna.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+created_by: Pablo Casal pcasal@gmail.com
```
Comment (jjasghar) [inline knowledge/astronomy/constellations/Phoenix/qna.yaml:1]: Please put your GitHub Username here.

File: knowledge/astronomy/constellations/Phoenix/qna.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+domain: constellation
+seed_examples:
+- answer: The constellation does not lie on the galactic plane of the Milky Way, and there are no prominent star clusters. Lying within the bounds of the constellation is the gigantic Phoenix cluster, which is around 7.3 million light years wide and 5.7 billion light years away. It is located in the southern sky. The constellation stretches from roughly −39° to −57° declination, and from 23.5h to 2.5h of right ascension. The constellations Phoenix, Grus, Pavo and Tucana, are known as the Southern Birds.
```
Comment (jjasghar) [inline knowledge/astronomy/constellations/Phoenix/qna.yaml:4]: Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please fix these issues then we can run the linter.

**Comment by jjasghar** [comment]
Closing per the combined PR here: https://github.com/instructlab/taxonomy/pull/1159

**Pull request closed without merging**